beacon_node/beacon_chain/src/early_attester_cache.rs

```diff
@@ -122,7 +123,7 @@ impl<E: EthSpec> EarlyAttesterCache<E> {
             item.committee_lengths
                 .get_committee_length::<E>(request_slot, request_index, spec)?;
 
-        let attestation = Attestation {
+        let attestation = Attestation::Base(AttestationBase {
```

Comment:
I haven't looked closely (not sure if you have) but is it worth adding a `TODO(electra)` here to consider refactoring this test to work with both forks?